pkg/controller/storagecluster/reconcile.go

```diff
@@ -639,29 +641,38 @@ func (r *ReconcileStorageCluster) deleteResources(sc *ocsv1.StorageCluster, reqL
 	return nil
 }
 
-// reconcileUninstallAnnotations looks at the current uninstall annotations on the StorageCluster and sets defaults if none are set.
+// reconcileUninstallAnnotations looks at the current uninstall annotations on the StorageCluster and sets to defaults if none or unrecognized one's are set.
```

Comment:
s/one's/ones

Reply:
Done

Reply:
"sets defaults" is correct - no need to add the "to".

Reply:
done